### DISCUSSION TITLE
evm-browser 无法安装成功

### DISCUSSION THREAD

**Discussion by linghuccc** [IntensiveCoLearning/Web3-URL#109] [Q&A] [answered]
git clone 了 evm-browser 之后, yarn install 了 dependency，但是在打开 dapp 的时候，yarn start 报错

![evm](https://github.com/user-attachments/assets/28f1a1a9-3f5e-4877-bbf7-100c28372686)

求教，这个错误怎么解决？

**Comment by HeliosLz** [comment, marked as answer]
直接下载安装包试试。

**Reply by linghuccc** [reply]
> 直接下载安装包试试。

谢谢，安装包在 window 系统里面可以使用。

**Comment by qzhodl** [comment]
可以给repo提一个issue

**Reply by linghuccc** [reply]
是的，我已经提过了，谢谢
